### PR TITLE
feat: contributor guidelines

### LINKED ISSUE #857: Add contributor guidelines

### PR DESCRIPTION
**Pull request by mekhlakapoor**
closes #857

### CODE REVIEW

**Review by dyf — CHANGES_REQUESTED** [review]
can you migrate relevant things from the "Contributing" section from README.md here and add a README.md -> CONTRIBUTION.md link?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+- Use **interrogate** to check that modules, methods, etc. have been documented thoroughly:
+
+  ```
+  interrogate .
+  ```
+  For more information you can run
+  ```interrogate --verbose .```
+
+- Use **flake8** to check that code is up to standards (no unused imports, etc.):
```
Comment (saskiad) [inline CONTRIBUTING.md:62]: Is there a link that we can point people to as to what the standards are?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
+# Contributor Guidelines
+This repository defines the schemas needed to ingest and validate metadata. As a core service, it is used across multiple teams and services. Therefore, any contributions must follow certain rules to ensure stability and organization.
+This document will go through best practices for contributing to this project
+
+## Issues and Feature Requests
+Feature requests and bug reports are all welcome as [issues](https://github.com/AllenNeuralDynamics/aind-data-schema/issues). Please use the provided [templates](https://github.com/AllenNeuralDynamics/aind-metadata-mapper/issues/new/choose) to ensure we have enough information to work with.
+If you'd like to propose a large change or addition, or generally have a question about how things work, head start a new [Discussion](https://github.com/AllenNeuralDynamics/aind-data-schema/discussions)!
```
Comment (saskiad) [inline CONTRIBUTING.md:7]: remove the work "head" ?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
+# Contributor Guidelines
+This repository defines the schemas needed to ingest and validate metadata. As a core service, it is used across multiple teams and services. Therefore, any contributions must follow certain rules to ensure stability and organization.
+This document will go through best practices for contributing to this project
+
+## Issues and Feature Requests
+Feature requests and bug reports are all welcome as [issues](https://github.com/AllenNeuralDynamics/aind-data-schema/issues). Please use the provided [templates](https://github.com/AllenNeuralDynamics/aind-metadata-mapper/issues/new/choose) to ensure we have enough information to work with.
```
Comment (saskiad) [inline CONTRIBUTING.md:6]: Can we ask people to not assign people in their tickets. Maybe say something about the process - eg. create a ticket and we will review it and address it in our sprint. If they want to grab someone's attention, they can tag us in the issue. But there are people who have been assigning tickets and it disrupts the process (imo).

File: CONTRIBUTING.md (hunk 2 above)
Comment (saskiad) [inline CONTRIBUTING.md:7]: I might phrase this as "If you have a broader suggestion or a question about how things work"

File: CONTRIBUTING.md (hunk 3 above)
Comment (saskiad) [inline CONTRIBUTING.md:6]: 'Feature request' means something to SWE that might not be the same as many of the scientists in our org. Is there a way to explain the scope of what a reasonable feature request is?

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
+# Contributor Guidelines
+This repository defines the schemas needed to ingest and validate metadata. As a core service, it is used across multiple teams and services. Therefore, any contributions must follow certain rules to ensure stability and organization.
```
Comment (saskiad) [inline CONTRIBUTING.md:2]: "ingest" isn't really right here... This isn't ingesting metadata, it's defining the schema for the metadata. 
"This repository defines the metadata schema used to document our data assets." Maybe? It does also validate... but I worry that saying ingest will imply that this repo is doing the mapping, which it is not.

**Review by dyf — APPROVED** [review]

**Pull request merged**